src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -433,6 +433,7 @@ function chip_tests_pics(options)
 
 async function chip_tests(list, options)
 {
+  initClusters.call(this);
```

Comment:
Shouldn't this have an await?  I guess the point is that it kicks off the async process that eventually makes assertCommandOrAttribute work, so we don't really need to await here?  It might be worth documenting that.